src/binutil/server.rs

```diff
@@ -322,6 +322,7 @@ fn run_raft_server(pd_client: RpcClient, cfg: &TiKvConfig, security_mgr: Arc<Sec
             pd_client,
             resolver,
             cfg.pessimistic_txn.monitor_membership_interval,
+            cfg.pessimistic_txn.wait_for_lock_timeout,
```

Comment:
I'm afraid there might be too many arguments. May it be better if we pass the whole `cfg.pessimistic_txn`?

Reply:
Yes. I will file a PR to pass `&cfg.pessimistic_txn` to `WaiterManger` and `Detector`.